src/MessageBroker/RabbitMQ/PHPAmqp/PHPAmqpConsumerAdapter.php

```diff
@@ -104,7 +104,7 @@ private function callback($message, callable $callback)
     {
         return $callback(new Message(
             $message->getBody(),
-            $message->get('timestamp'),
+            date('c', $message->get('timestamp')),
```

Comment:
It's a date string in `ISO8601` format, I thought that we should just keep the constructor params as they are, but I have changed them all to be `DateTime`.